src/ITfoxtec.Identity.Saml2/Schemas/Saml2SecurityAlgorithms.cs

```diff
@@ -38,5 +38,6 @@ public static class Saml2SecurityAlgorithms
         /// </summary>
         public const string RsaSha512Signature = "http://www.w3.org/2001/04/xmldsig-more#rsa-sha512";
 
```

Comment:
A comment like for the other constants would be nice

Reply:
added comment